test/transaction.test.js

```diff
@@ -6,13 +6,16 @@
 'use strict';
 var Transaction = require('../index').Transaction;
 
-var expect = require('chai').expect;
+const chai = require('chai');
+const expect = chai.expect;
+const chaiAsPromised = require('chai-as-promised');
 var testConnector = require('./connectors/test-sql-connector');
 
 var juggler = require('loopback-datasource-juggler');
 var db, Post, Review;
 describe('transactions', function() {
   before(function(done) {
+    chai.use(chaiAsPromised);
```

Comment:
Please move this line to the top, where `chaiAsPromised` is loaded.

Personally, I would write this as follows:

```ts
const chai = require('chai');
chai.use(require('chai-as-promised'));
const {expect} = chai;
```